scipy/stats/mstats_extras.py

```diff
@@ -392,7 +392,50 @@ def compare_medians_ms(group_1, group_2, axis=None):
         ndarray of floats with a length equal to the length of `group_1`
         along `axis`.
 
+    Examples
+    --------
+    >>> from scipy import stats
+    >>> a = [1, 2, 3, 4, 5, 6, 7]
+    >>> b = [8, 9, 10, 11, 12, 13, 14]
+
+    Arrays must be at least of size 7.
+
+    >>> stats.mstats.compare_medians_ms(a, b, axis=None)
+    1.0693225866553746e-05
+
+    Since no axis is provided, the result is flattened.
```

Comment:
```suggestion
```
When no axis is provided, the input arrays are flattened. Since they are already flat (1D), this has no effect. So there is no need to write this here, as the example is not demonstrating the effect of this.